model/src/main/kotlin/config/RepositoryConfiguration.kt

```diff
@@ -68,5 +68,7 @@ data class RepositoryConfiguration(
      * Defines license choices within this repository.
      */
     @JsonInclude(value = JsonInclude.Include.CUSTOM, valueFilter = LicenseChoicesFilter::class)
-    val licenseChoices: LicenseChoices = LicenseChoices()
+    val licenseChoices: LicenseChoices = LicenseChoices(),
+
+    val packageSnippetChoices: List<PackageSnippetChoice> = emptyList()
```

Comment:
This should have `@JsonInclude(value = JsonInclude.Include.NON_EMPTY)` (see above).